packages/@react-spectrum/tree/docs/TreeView.mdx

```diff
@@ -0,0 +1,512 @@
+import FileTxt from '@spectrum-icons/workflow/FileTxt';
+import GlobeOutline from '@spectrum-icons/workflow/GlobeOutline';
+import Image from '@spectrum-icons/workflow/Image';
+import {Text} from '@react-spectrum/text';
+import {TreeView, TreeViewItem} from '@react-spectrum/tree';
+import {JSX} from "react";
+import {Key} from "@react-types/shared";
+```
+
+---
+category: Collections
+keywords: [tree, grid]
+---
+
+# TreeView
+
+<PageDescription>{docs.exports.TreeView.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['TreeView', 'TreeViewItem']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/tree-view/'}
+  ]} />
+
+## Example
+
+```tsx example
+<TreeView aria-label="Example tree with static contents" defaultExpandedKeys={new Set(['documents', 'photos'])} height="size-4600">
+  <TreeViewItem id="documents" textValue="Documents">
+    <Text>Documents</Text>
+    <Folder />
+    <TreeViewItem id="project-a" textValue="Project A">
+      <Text>Project A</Text>
+      <Folder />
+      <TreeViewItem id="weekly-report" textValue="Weekly-Report">
+        <Text>Weekly Report</Text>
+        <FileTxt />
+      </TreeViewItem>
+    </TreeViewItem>
+    <TreeViewItem id="document-1" textValue="Document 1">
+      <Text>Document 1</Text>
+      <FileTxt />
+    </TreeViewItem>
+    <TreeViewItem id="document-2" textValue="Document 2">
+      <Text>Document 2</Text>
+      <FileTxt />
+    </TreeViewItem>
+  </TreeViewItem>
+  <TreeViewItem id="photos" textValue="Photos">
+    <Text>Photos</Text>
+    <Folder />
+    <TreeViewItem id="image-1" textValue="Image 1">
+      <Text>Image 1</Text>
+      <Image />
+    </TreeViewItem>
+    <TreeViewItem id="image-2" textValue="Image 2">
+      <Text>Image 2</Text>
+      <Image />
+    </TreeViewItem>
+    <TreeViewItem id="image-3" textValue="Image 3">
+      <Text>Image 3</Text>
+      <Image />
+    </TreeViewItem>
+  </TreeViewItem>
+</TreeView>
+```
+
+## Content
+TreeView is a [collection component](../react-stately/collections.html) that provides users with a way to navigate nested hierarchical information.
+
+Basic usage of TreeView, seen in the example above, shows the use of a static collection where the contents of the TreeView are hard coded. Dynamic collections, as shown below, can be used when the options come from an external data source, such as an API, or update over time. Providing the data in this way allows TreeView to automatically cache the rendering of each item, which dramatically improves performance.
+
+Items can be statically defined as children, or generated dynamically using a function based on the data passed to the `items` prop.
+
+Each item has a unique key defined by the data. The `key` of each item element is implicitly defined by the id property of the item object. See [collections](../react-stately/collections.html#unique-keys) to learn more keys in dynamic collections.
```

Comment:
while relevant, this technically points to the wrong collection implementation. We should point this to https://react-spectrum.adobe.com/react-aria/collections.html#unique-ids instead

Reply:
Thanks for catching these!